arb-gateway/src/ArbProofService.ts

```diff
@@ -2,6 +2,7 @@
 import { EVMProofHelper, type IProofService } from '@ensdomains/evm-gateway';
 import { AbiCoder, Contract, EventLog, ethers, toBeHex, type AddressLike, toNumber } from 'ethers';
 import rollupAbi from "./abi/rollupABI.js";
+import type { IBlockCache } from './cache/IBlockCache.js';
```

Comment:
I think you forgot to commit this file.

Reply:
yes it was part of .gitignore. It is committed now